ansible/roles/django/templates/settings_secret.py.j2

```diff
@@ -0,0 +1,16 @@
+from requests.auth import HTTPBasicAuth
+
+SECRET_KEY = '{{ secret.secret_key }}'
+KRATOS_ADMIN_PWD = '{{ secret.couch_pwd }}'
+
+DATA_DIR = '{{ data_dir }}/django'
+
+SOCIAL_AUTH_GITHUB_ENTERPRISE_KEY = '{{ github_enterprise.OAUTH.KEY }}'
```

Comment:
This is an excellent point, and that's what we are doing with kratos/moirai. However I couldn't figure out an even semi-elegant way to actually do it. I think this is definitely something we should revisit. I've created an issue.